app/commands/concerns/equity_composite_index_admin_fixes.rb

```diff
@@ -0,0 +1,24 @@
+require 'active_support/concern'
+
+module EquityCompositeIndexAdminFixes
```

Comment:
Suggestion: Since we have the opportunity to use a specific name here, I think we should avoid the suffix `Fixes` in favor of something more explicit about the behavior we're adding w/ the concern. I'd recommend we rename `EquityQuintilePresenterFixes` while making that change, since we're adopting this pattern more. :)

Reply:
More of a nitpick than a suggestion 😰